packages/at_persistence_secondary_server/test/commit_log_test.dart

```diff
@@ -840,6 +840,33 @@ void main() async {
         expect(commitEntriesMap.containsKey('public:phone.wavi@alice'), false);
         expect(commitEntriesMap.containsKey('public:location@alice'), true);
       });
```

Comment:
Please add another test which both uses a regex and sets skipDeletes: true

Reply:
Done